src/MetaArrays.jl

```diff
@@ -99,11 +99,19 @@ a meta array of `T`.
 """
 const MetaUnion{T} = Union{MetaArray{<:T},T}
 
-function Base.show(io::IO,::MIME"text/plain",x::MetaArray) where M
+function Base.show(io::IO, ::MIME"text/plain", x::MetaArray)
```

Comment:
unused type parameters sometimes have performance penalties, so better to avoid them. Not that it matters particularly for `show`

Reply:
Thanks for the advice!